gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImplCreateTest.java

```diff
@@ -248,7 +243,7 @@ public void testCreateObjectApiInterruptedException() throws Exception {
         .thenReturn(
             new FakeWriteChannel() {
               @Override
-              public int write(ByteBuffer src) {
```

Comment:
1. Is it really need to be handed over to `close` function. 
1. Given we don't have pip anymore for javastorage writer, this test seems to be an overkill for javastorage. Should be bypass it?